tools/run_tests/performance/scenario_config.py

```diff
@@ -127,7 +128,12 @@ def _ping_pong_scenario(name, rpc_type,
     'warmup_seconds': warmup_seconds,
     'benchmark_seconds': BENCHMARK_SECONDS
   }
-  if use_generic_payload:
+  if use_big_generic_payload:
```

Comment:
nit: ideally, if both big_generic_payload and use_generic_payload are set, this should throw so we don't get unexpected behavior.
Fine to leave for now I think.